aicodebot/lm.py

```diff
@@ -84,46 +83,17 @@ def model_factory(
                 streaming=streaming,
                 callbacks=callbacks,
             )
-        elif provider == self.HUGGINGFACE_HUB:
-            return self.get_huggingface_hub_model(
+        elif provider == self.OLLAMA:
+            return self.get_ollama_model(
                 model_name,
                 response_token_size=response_token_size,
                 temperature=temperature,
-                streaming=streaming,
+                # streaming=streaming,
```

Comment:
Ollama doesnt have a streaming parameter and is instead replaced with the 'callbacks' parameter. I'll make note of this because it might change how Ollama is handled elsewhere in the codebase.